x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/transport/netty4/SecurityNetty4HttpServerTransportCloseNotifyTests.java

```diff
@@ -251,7 +251,7 @@ public void close() {
             server.dispatcher.reqQueue.forEach(r -> r.request.getHttpRequest().release());
             server.netty.stop();
             server.threadPool.shutdownNow();
-            safeAwait(client.netty.config().group().shutdownGracefully());
```

Comment:
Should we fix the other `shutdownGracefully()` calls with default timeouts? Maybe in a follow-up?